Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/MoreInformationView.kt

```diff
@@ -69,7 +69,7 @@ class MoreInformationView @JvmOverloads constructor(
         val subtitle = binding.subtitleElement.text
         val isToggleVisible = binding.toggle.isVisible
         contentDescription = if (isToggleVisible) {
-            "$title $subtitle $isChecked"
+            "$title $subtitle"
```

Comment:
now as both branches of if statement do the same it could be removed from code and set in xml

Reply:
Yes, that's right. I'll change it.